tbd.schema.json

```diff
@@ -61,17 +61,12 @@
       "uniqueItems": true
     },
     "building": {
-      "type": "array",
-      "items": {
-        "$ref": "#/definitions/Building"
-      },
-      "uniqueItems": true
+      "type": "object",
+      "$ref": "#/definitions/Building"
     },
-    "logs": {
-      "type": "array",
-      "items": {
-        "$ref": "#/definitions/Log"
-      }
+    "log": {
```

Comment:
The test output I saw had `logs` instead of `log`

Reply:
Fixed.